python/cudf/cudf/core/column/column.py

```diff
@@ -121,10 +122,13 @@ def __len__(self):
         return self.size
 
     def to_pandas(self, index=None, **kwargs):
-        pd_series = self.to_arrow().to_pandas(**kwargs)
+        if str(self.dtype) in NUMERIC_TYPES and self.null_count == 0:
+            pd_series = pd.Series(cupy.asnumpy(self.values))
```

Comment:
Can we confirm this works for `datetime` and `timedelta` types as well? CuPy doesn't support them...

Reply:
It works - 

```
>>> x = cudf.Series([1,2,3], dtype='datetime64[us]')                                                                                                                                                                                                                       
>>> cp.asnumpy(x._column.values)                                                                                                                                                                                                                                           
array(['1970-01-01T00:00:00.000001', '1970-01-01T00:00:00.000002',                                                                                                                                                                                                         
       '1970-01-01T00:00:00.000003'], dtype='datetime64[us]')                                                                                                                                                                                                              
>>> x._column.values                                                                                                                                                                                                                                                       
array(['1970-01-01T00:00:00.000001', '1970-01-01T00:00:00.000002',                                                                                                                                                                                                         
       '1970-01-01T00:00:00.000003'], dtype='datetime64[us]')
>>> type(x._column.values)
<class 'cupy.core.core.ndarray'>
>>> x._column.values.dtype
dtype('<M8[us]')
```

So it's supported on some level. It looks like `values` produces a cupy array regardless of dtype, so if we want to move away from this we should probably fix it there as well.